user-interface/src/main/java/life/qbic/datamanager/views/account/PersonalAccessTokenComponent.java

```diff
@@ -137,6 +140,34 @@ private void showCreatedPersonalAccessToken(String rawTokenText) {
     updateUI();
   }
 
+private void handleCopyClicked(String rawTokenText) {
```

Comment:
this effectively prevents users from copying the token multiple times. This is bad IMHO because if I accidentally overwrite it in my clipboard I lost the ability to copy it easily. How about leaving the copy icon in place and only changing text and background color?